tests/tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Streisand syntax check.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export ANSIBLE_CONFIG=$DIR/ansible.cfg
+
+set -e
+
+command -v ansible > /dev/null 2>&1 || {
+  echo "Ansible is not installed."
+  exit 1
+}
+
+required_ansible_version="2.3.0.0"
+
+if [[ "$(ansible --version | grep -oe '2\(.[0-9]\)*')" < $required_ansible_version ]]
+then
+    echo "Ansible $required_ansible_version or higher is required."
+    exit 1
+fi
```

Comment:
Is it possible to abstract out the common parts of this and the main streisand shell script so that it can be sourced here? It would be nice to not have to keep the `required_ansible_version` in sync for one example.